recipe/meta.yaml

```diff
@@ -46,12 +45,92 @@ test:
     - PyHyperScattering
   commands:
     - pip check
-
+outputs:
+  - name: pyhyperscattering-base
+    build:
+      script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+    run_exports:
+      # keep downstream ABI-compatible on minor bumps
+      - {{ pin_subpackage("pyhyperscattering-base", max_pin="x.x") }}
```

Comment:
It's not usual to have `run_exports` on plain python packages. You shouldn't need to recompile a downstream package if this changes, right?